CMakeLists.txt

```diff
@@ -149,6 +151,15 @@ if(FV3)
   add_subdirectory(FV3)
 endif()
 
+###############################################################################
```

Comment:
Following @aerorahul 's suggestion, this would become
```
#################################################################################
# Chemistry Components [GOCART]
###############################################################################
if(GOCART)
  add_subdirectory(GOCART-interface)
endif()
```

Reply:
See previous comment.

Reply:
Resolved via internal discussions.